packages/sdk/src/credential/m365TenantCredential.ts

```diff
@@ -48,16 +49,24 @@ export class M365TenantCredential implements TokenCredential {
 
     const config = this.loadAndValidateConfig();
 
-    const tokenCredentialOptions: TokenCredentialOptions = {
-      authorityHost: config.authorityHost,
-    };
-
-    this.clientSecretCredential = new ClientSecretCredential(
-      config.tenantId!,
-      config.clientId!,
-      config.clientSecret!,
-      tokenCredentialOptions
-    );
+    if (config.certificatePath) {
```

Comment:
If user both provide certificatePath and clientSecret, do we need to give some tips/or throw error that we will use certificate by default?